astropy/visualization/wcsaxes/tests/test_formatter_locator.py

```diff
@@ -456,6 +458,15 @@ def test_incompatible_unit_decimal(self):
         ):
             AngleFormatterLocator(unit=u.arcmin, decimal=False)
 
+    @pytest.mark.parametrize("cls", [AngleFormatterLocator, ScalarFormatterLocator])
+    def test_unicode_minus(self, cls):
+        # see https://github.com/astropy/astropy/issues/15898
+        fl = cls()
+        with rc_context(rc={"axes.unicode_minus": True}):
+            minus_one, _ = fl.formatter([-1.0, 1.0] * u.deg, spacing=2 * u.deg)
+
+        assert minus_one.startswith(unicodedata.lookup("MINUS SIGN"))
```

Comment:
Could just use "\N{MINUS SIGN}" here

Reply:
ow, I didn't know about that ! I'll see what other reviewers think about it first, as I'm not sure how obscure this feature is and I'm worried it wouldn't be as readable to maintainers.